storage.py

```diff
@@ -17,6 +17,14 @@ def remove(self):
 
     def set(self):
         pass
-    
-    def add(self):
-        pass
\ No newline at end of file
+
+    def add(self, key, value):   #Катя
+        key = key
+        value = value
+
+        if key in self.data.keys():
```

Comment:
It is unnecessary to use method _keys()_ because it iterates through keys by default. For example:

```
if key in self.data:
    pass
```

Reply:
fixed it 👍